types/mongodb/index.d.ts

```diff
@@ -497,6 +498,11 @@ export interface Collection<TSchema = Default> {
     count(query: Object, callback: MongoCallback<number>): void;
```

Comment:
Can you add an `@deprecated` to the description of count on line 495 so the deprecated status if reflected here?

Reply:
Done. I've added a @deprecated sign per comment, under the URL to be consistent with insertOne, remove, etc. Thanks.

Reply:
Updated. cheers.